### PR TITLE
Hw 5/relational database

### PR DESCRIPTION
**Pull request by alchemixXx**
I didn't solve tasks 8 and 16. In task 16 made almost proper order (187 from 194), but couldn't do it completely.

### CODE REVIEW

File: relational_database/homework.py [hunk 1]
```diff
@@ -150,7 +181,9 @@ def task_12_list_suppliers_from_specified_countries(cur):
 
     Returns: 8 records
     """
-    pass
+    cur.execute("""SELECT * FROM suppliers 
+                   WHERE country = 'USA'  AND country='UK' AND country='Japan';""")
```
Comment (dimigor) [inline relational_database/homework.py]: You might use operator IN ('USA', 'UK' ....)

File: relational_database/tests.py [hunk 2]
```diff
@@ -91,72 +91,72 @@ def tearDownClass(cls) -> None:
...
-
-    def test_task_7(self):
-        with self.conn.cursor(cursor_factory=RealDictCursor) as cursor:
-            actual_result = task_7_list_supplier_countries_in_desc_order(cursor)
-            actual_result = [dict(record) for record in actual_result]
-            expected_result = self.load_rows_from_file("task_7.json")
-
-        for i, row in enumerate(actual_result):
-            self.assertDictEqual(row, expected_result[i])
+    # def test_task_1(self):
```
Comment (dimigor) [inline relational_database/tests.py]: Why you commented this tests?

File: relational_database/homework.py (hunk 1 above)
Comment (alchemixXx) [reply relational_database/homework.py]: OK, done. added specified fields in select and changed operator where.

File: relational_database/tests.py (hunk 2 above)
Comment (alchemixXx) [reply relational_database/tests.py]: sorry, it was just for debugging. 
task #9 was fixed.

**Review by dimigor — APPROVED** [review]

**Pull request merged**